src/mapbox/Colors.js

```diff
@@ -2,7 +2,19 @@
  * Variables referring to colors.
  */
 export const highlight = "#fff";
-export const countyFill = "#059713";
+export const countyFill = "#fff";
 export const countyOutline = "#124c1b";
 export const zipcodeFill = "#F3AD1C";
-export const zipcodeOutline = "#a4720b";
\ No newline at end of file
+export const zipcodeOutline = "#a4720b";
+export const colorMapFill = [
```

Comment:
Would an array of objects be a better way to store the colors. For example: { 'green': '#00FF00' }